### ISSUE TITLE
(cli): npm install fails the first time

### ISSUE THREAD

**Issue by SoManyHs** [aws/aws-cdk#14690]
### Reproduction Steps
```
$ npm install -g aws-cdk

removed 193 packages, changed 1 package, and audited 3 packages in 4s

found 0 vulnerabilities

$ cdk --version
internal/modules/cjs/loader.js:796
    throw err;
    ^

Error: Cannot find module 'source-map-support/register'
Require stack:
- /usr/local/lib/node_modules/aws-cdk/bin/cdk.js
- /usr/local/lib/node_modules/aws-cdk/bin/cdk
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)
    at Function.Module._load (internal/modules/cjs/loader.js:686:27)
    at Module.require (internal/modules/cjs/loader.js:848:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/aws-cdk/bin/cdk.js:4:1)
    at Module._compile (internal/modules/cjs/loader.js:955:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
    at Module.load (internal/modules/cjs/loader.js:811:32)
    at Function.Module._load (internal/modules/cjs/loader.js:723:14)
    at Module.require (internal/modules/cjs/loader.js:848:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/local/lib/node_modules/aws-cdk/bin/cdk.js',
    '/usr/local/lib/node_modules/aws-cdk/bin/cdk'
  ]
}
```

running `npm install -g aws-cdk` again seems to sort it out, but this has been happening to me for over a week.

### What did you expect to happen?

CDK executable installs smoothly

### What actually happened?

Running CDK commands fails with above error

### Environment

  - **CDK CLI Version  :** 1.103.0 (build bc13a66)
  - **Framework Version:**
  - **Node.js Version:**  v12.15.0
  - **OS               :** OSX

--- 

This is :bug: Bug Report

**Comment by ericzbeard** [comment]
Happens to me too, every time I do an install.

**Comment by rix0rrr** [comment]
I've just been installing the CDK a couple of times today and haven't seen this at all.

What's your NPM version?

**Comment by ericzbeard** [comment]
7.8.0

**Comment by rix0rrr** [comment]
Having a hard time reproducing this in a Docker container:

```
$  npm install -g npm@7.8.0
```

Been running this a couple of times, no failures:

```
$ npm uninstall -g aws-cdk && npm install -g aws-cdk && cdk --version
```

Can you go look at the `node_modules` tree of where the CLI is installed next time it happens? 

Perhaps run:

```
$ tree -d /usr/local/lib/node_modules
```

?

**Comment by SoManyHs** [comment]
```
$ npm -v
7.10.0

$ node -v
v12.15.0

$ uname -v
Darwin Kernel Version 19.6.0: Mon Apr 12 20:57:45 PDT 2021; root:xnu-6153.141.28.1~1/RELEASE_X86_64
```

**Comment by ericzbeard** [comment]
```
$ npm uninstall -g aws-cdk && npm install -g aws-cdk && cdk --version
```

What if you don't uninstall first?

**Comment by rix0rrr** [comment]
Are you using NVM by any chance?

**Comment by rix0rrr** [comment]
Eric has tried this:

```
$ npm install -g npm@6

# This throws errors occasionally
$ npm uninstall -g aws-cdk && npm install -g aws-cdk && cdk --version

npm ERR! code ENOENT
npm ERR! syscall rename
npm ERR! path /Users/ezbeard/.nvm/versions/node/v12.18.0/lib/node_modules/aws-cdk/node_modules/@tootallnate/once
npm ERR! dest /Users/ezbeard/.nvm/versions/node/v12.18.0/lib/node_modules/aws-cdk/node_modules/@tootallnate/.once.DELETE
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, rename '/Users/ezbeard/.nvm/versions/node/v12.18.0/lib/node_modules/aws-cdk/node_modules/@tootallnate/once' -> '/Users/ezbeard/.nvm/versions/node/v12.18.0/lib/node_modules/aws-
```

**Comment by SoManyHs** [comment]
I'm not using nvm, still seeing this.

**Comment by mbonig** [comment]
I am using nvm and I'm seeing this.

**Comment by rix0rrr** [comment]
I wonder if it might be this bug: https://github.com/npm/cli/issues/3196

Which I *think* should have been fixed by this: https://github.com/npm/arborist/commit/2ead1e8588cc0cb424e577de9fc561f47d3eb8ab

Released in `@npm/arborist 2.6.1`, which should be in NPM 7.15.1.

Can you give upgrading to 7.15.1 a shot and then retrying? (Might be worth a clean install as well, just to be sure...)
